.yarnrc.yml

```diff
@@ -0,0 +1,3 @@
+nodeLinker: node-modules
+
+yarnPath: .yarn/releases/yarn-4.5.0.cjs
```

Comment:
This and the binary file before this should've been merged in https://github.com/fossadev/fossabot-docs/pull/80

whoops